### PR TITLE
[WIP] Sprout - Retrieve and show providers managed by appliances

### PR DESCRIPTION
**Pull request by mfalesni**
WIP for no PR testing.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8f3f7b06441a1c879fed39504444bb6362873654:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: utils/appliance.py [hunk 1]
```diff
@@ -352,6 +352,23 @@ def __enter__(self):
...
+        """
+        ems_table = self.db["ext_management_systems"]
+        ip_addresses = set([])
+        for ems in self.db.session.query(ems_table):
+            ip_addresses.add(ems.ipaddress)
+        provider_keys = set([])
+        for provider_key, provider_data in conf.cfme_data.get("management_systems", {}).iteritems():
+            if provider_data.get("ipaddress", None) in ip_addresses:
+                provider_keys.add(provider_key)
+        return provider_keys
```
Comment (psav) [inline utils/appliance.py]: Does sprout pick up on cfme_data yaml changes?

Comment (mfalesni) [inline utils/appliance.py]: Not yet, but I can do reload it without others noticing.

**Comment by mfalesni** [comment]
OK merging then and updating sprout.

**Pull request merged**